sdk/boms/azure-spring-boot-bom/pom.xml

```diff
@@ -81,22 +85,27 @@
       <dependency>
         <groupId>com.azure.spring</groupId>
         <artifactId>azure-spring-boot-starter-cosmos</artifactId>
+        <version>${azure.spring.boot.cosmos.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>com.azure.spring</groupId>
+        <artifactId>azure-spring-boot-starter-keyvault-certificates</artifactId>
         <version>${azure.spring.boot.version}</version>
```

Comment:
I thought the version for this is 3.**0**.1?

Reply:
Updated. Thank you for carefully check.